pkg/apis/forklift/v1beta1/plan/vm.go

```diff
@@ -30,6 +30,8 @@ type VM struct {
 	ref.Ref `json:",inline"`
 	// Enable hooks.
 	Hooks []HookRef `json:"hooks,omitempty"`
+	// Choose the bootable disk number for praimery
```

Comment:
primary? we should also say that this field is optional and what happens when it's not specified
```suggestion
                      description: Choose the primary disk the VM boots from
```